docs/output.md

```diff
@@ -622,3 +626,17 @@ This custom metadata is presented as a data table in the MultiQC report file.
 </details>
 
 [Nextflow](https://www.nextflow.io/docs/latest/tracing.html) provides excellent functionality for generating various reports relevant to the running and execution of the pipeline. This will allow you to troubleshoot errors with the running of the pipeline, and also provide you with other information such as launch commands, run times and resource usage.
+
+### nf-core/proteinfold
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `proteinfold/`
+  - `<samplename>/`
+    - `<id>.faa`: Amino acid file(s) containing one family representative sequence each.
+  - `samplesheet_<samplename>.csv`: Downstream samplesheet to be used as nf-core/proteinfold input.
+
+</details>
+
+[nf-core/proteinfold](https://nf-co.re/proteinfold) is a bioinformatics best-practice analysis pipeline for Protein 3D structure prediction.
```

Comment:
I would document a bit more what is in the samplesheet, so a user knows a bit more what is going into the samplesheet and how it can be used with proteinfold (I guess proteinfold has lots of options)